Content.Server/DeviceLinking/Systems/PressurePlateSystem.cs

```diff
@@ -0,0 +1,98 @@
+using Content.Server.DeviceLinking.Components;
+using Content.Shared.DeviceLinking;
+using Content.Server.Storage.Components;
+using Content.Shared.Placeable;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Events;
+
+namespace Content.Server.DeviceLinking.Systems;
+
+/// <see cref="PressurePlateComponent"/>
+public sealed class PressurePlateSystem : EntitySystem
+{
+    [Dependency] private readonly DeviceLinkSystem _signalSystem = default!;
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<PressurePlateComponent, ComponentInit>(OnInit);
+        //SubscribeLocalEvent<PressurePlateComponent, EndCollideEvent>(OnEndCollide);
+        //SubscribeLocalEvent<PressurePlateComponent, StartCollideEvent>(OnStartCollide);
+        SubscribeLocalEvent<PressurePlateComponent, ItemPlacedEvent>(OnItemPlaced);
+        SubscribeLocalEvent<PressurePlateComponent, ItemRemovedEvent>(OnItemRemoved);
+    }
+
+    private void OnItemRemoved(EntityUid uid, PressurePlateComponent component, ref ItemRemovedEvent args)
+    {
+        UpdateState(uid, component);
+    }
+
+    private void OnItemPlaced(EntityUid uid, PressurePlateComponent component, ref ItemPlacedEvent args)
+    {
+        UpdateState(uid, component);
+    }
+
+    private void OnInit(EntityUid uid, PressurePlateComponent component, ComponentInit args)
+    {
+        _signalSystem.EnsureSourcePorts(uid, component.PressedPort, component.ReleasedPort);
+        _appearance.SetData(uid, PressurePlateVisuals.Pressed, false);
+    }
+    //private void OnStartCollide(EntityUid uid, PressurePlateComponent component, ref StartCollideEvent args)
+    //{
+    //    UpdateState(uid, component);
+    //}
+    //
+    //private void OnEndCollide(EntityUid uid, PressurePlateComponent component, ref EndCollideEvent args)
+    //{
+    //    UpdateState(uid, component);
+    //}
+
```

Comment:
```suggestion
```

just remove, dont comment out. you can leave a comment somewhere explaining that this used to use collisions or something